core/src/main/scala/org/apache/spark/rdd/DoubleRDDFunctions.scala

```diff
@@ -74,6 +74,20 @@ class DoubleRDDFunctions(self: RDD[Double]) extends Logging with Serializable {
   }
 
   /**
+   * Compute the population standard deviation of this RDD's elements.
+   */
+  def popStdev(): Double = self.withScope {
```

Comment:
Oh, I forgot: these need `@Since` annotations. Hm, I'm not clear whether we should merge this for 2.0.0 now. I wouldn't mind but we have an RC now. It's not super urgent. I might mark this as since 2.1.0.